notebooks/237-segment-anything/237-segment-anything-fiftyone.ipynb

```diff
@@ -0,0 +1,2331 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import fiftyone as fo
</pre><p>On Ubuntu 22.04, this import fails due to https://github.com/voxel51/fiftyone/issues/1803.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>